trusted-committer-article.md

```diff
@@ -0,0 +1,232 @@
+  community is a precondition for healthy code. This is why the Apache Software
+  Foundation, after which many InnerSource initiative were modeled, has the
+  motto "Community over Code".
+- It is this potential area of conflict, where the TC plays a vital role.
+- One example I remember is when I developed a system for remotely controlling
+  HVAC systems in my InnerSource community. We successfully met the deadline to
+  present the system on a trade fair, where it was an overall success. But we
+  knew that we still had to iron out a couple of wrinkles, before we could
+  launch the system into the market.  When the planned time for market
+  introduction came closer and we were reasonably sure that we would not be
+  ready in time, it was my responsibility as a TC to engage with management,
+  explain the risks of launching before we were ready and to convince them to
+  delay the introduction for a couple of days. These were difficult
+  conversations but they were necessary to protect the community from the
+  backlash we would have gotten, if our system would have performed
+  suboptimally in the market. Because a successful market introduction really
+  was the goal we were all striving towards and taking that away would have
+  been a huge blow to the communities motivation.
+- Another common example is when contributors are not given time or permission
+  to make a contribution, especially when the community is working on a 
+  product which is not part of the contributors departments domain. In this case,
+  the TC should engage in discussion with the contributors manager and lobby for
+  an alternative decision.
+- Other examples 
+    - Advocating the need for required hardware/software 
+    - Making sure the community gets public credit for their work
+    - ...
+
+## Becoming a Trusted Committer
+
+- Open source principles of transparency and meritocracy guide the process by
+  which contributors can become TCs.
+- I have made it a point to only then nominate contributors as TCs when they
+  have proven their technical merit AND the social merit. In other words, they
+  must have made sort of social contributions, such as e. g. 
+    - helping onboarding new contributors
+    - organizing community events
+    - support community members in forums or mailing lists
+    - representing the community on events
+- In my community, the current TCs made the decision to officially ask a 
+  Contributor to assume the TC role, which is of course, voluntary.
+- There might well be multiple TCs in one project
+- TCs might step down, when they no longer have the time to perform the duties
+  of a TC.
+
+## Is the Trusted Committer a role for you?
+
+- demonstrated technical competence to maintain rank in the meritocracy
+- ability to communicate with peers, with product owners and with management
+- emotional maturity to deal with stressful social situations
+- sufficient standing in the organization to effectively advocate the
```

Comment:
Again not sure if this is a requirement as you could have an awesome trusted committer that is just helping uplevel community members and looking out for PRs while some other person (PO?) is doing this advocacy role.

Reply:
Problem here might be that I was personally in both the TC and PO role at the same time in my community. That said, if we define _advocacy for the communities needs_ as one of the TCs responsibilities, she would need this standing, or else she really can't effectively assume that responsibility, can she, @rrrutledge and @mrsanz ?

Reply:
How would you suggest we rephrase, @rrrutledge ? Do you want to distinguish between _core_ abilities and _nice to have_ ones?

Reply:
Sure, that works.

Reply:
By the way I think it is somewhat common as a grass-roots project starts out to have a single person serve as both TC and PO.  But it's also possible to split out the roles and larger projects would tend to have that happen so it's good to be clear about which responsibilities are primarily of one person or the other.